Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12-alpine
```

Comment:
should we try 14?

Reply:
Yes, I took the version from the `tokenization-service`.